deps/oblib/src/lib/net/ob_net_util.cpp

```diff
@@ -77,6 +77,13 @@ int ObNetUtil::get_local_addr_ipv6(const char *dev_name, char *ipv6, int len,
       } // if end
       ifa_tmp = ifa_tmp->ifa_next;
     } // while end
+
+    // if dev_name is invalid, than level will keep its initial value(-1)
```

Comment:
than -> then?

Reply:
Yes, it should be [then].

Reply:
> than -> then?

I have fixed the spelling error by changing [than] to [then].